packages/browser/package.json

```diff
@@ -45,7 +45,7 @@
     "testEnvironment": "jsdom"
   },
   "scripts": {
-    "build": "tsc -p tsconfig.json && esbuild src/index.ts --bundle --sourcemap --target=es2017 --format=esm --outfile=dist/index.bundle.js && esbuild src/index.ts --bundle --minify --sourcemap --target=es2019 --format=esm --outfile=dist/index.bundle.min.js",
```

Comment:
Why did you downgrade this one from es2019 to es2017?

Reply:
Because all the other ones were es2017. I'm going to upgrade all of these.